### ISSUE TITLE
The navigation should be scrollable

### ISSUE THREAD

**Issue by lauraponce** [18F/cms-hitech-apd#1475]
When the left nav extends below the visible page (usually happens when multiple sections are expanded), there is no way to scroll down within the nav and select a later section.

![Screen Shot 2019-04-17 at 2.38.19 PM.png](https://images.zenhubusercontent.com/57f4106595e42b6c5ba9e43c/10318445-abd1-4cb2-99ff-b82584fd1235)

### This task is done when...
 - [ ] The nav is scrollable independent of the page scroll

**Comment by NAretakis** [comment]
I think this is a duplicate of #1432 @lauraponce if you think they are substantially different issues, otherwise we I think we can close this one.

**Comment by lauraponce** [comment]
I think they are related, but this one seems more like a bug since users should always be able to scroll, while #1432 is more about the selected item and not necessarily the scroll behavior. It could be the case that something happens in #1432 that fixes this issue, but I think it makes sense to keep this open for now.
